### PR TITLE
Make Kara's suggested fixes to github actions page.

### PR DESCRIPTION
**Pull request by jonathanmetzman**

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
Still TODO:
1. Screenshots.
2. Dealing with storage repo better.
3. Dealing with GITHUB_TOKEN better (maybe just put this everywhere).

File: docs/running-clusterfuzzlite/github_actions.md [hunk 1]
```diff
@@ -13,25 +13,32 @@ permalink: /running-clusterfuzzlite/github-actions/
 {:toc}
 ---
 
+This page explains how to set up ClusterFuzzLite to run on GitHub Actions.
+To get the most of this page, you should have already set up your
+[build integration] and read the more
+[high-level document on running ClusterFuzzLite]
```
Comment (olivekl) [inline docs/running-clusterfuzzlite/github_actions.md:19]: Period needed

Comment (jonathanmetzman) [reply docs/running-clusterfuzzlite/github_actions.md]: Done.

**Comment by jonathanmetzman** [comment]
Kara, if you have any extra comments and are waiting to finish the review, feel free to submit them now so I can get started.

File: docs/running-clusterfuzzlite/github_actions.md [hunk 2]
```diff
@@ -245,61 +245,115 @@ on:
...
+You can copy and paste the above file and pruning should just work.
+But here is where `storage-repo`, `storage-repo-branch`, and
+`storage-repo-branch-coverage` come in handy.
+
+If `storage-repo` is set and `storage-repo-branch-coverage` is "gh-pages" (the
+default), then coverage reports can be viewed at
+`https://USERNAME.github.io/STORAGE-REPO-NAME/coverage/latest/report/linux/report.html`.
+This should be much nicer than downloading the coverage report as an artifact.
+
+### Extra configuration
```
Comment (oliverchang) [inline docs/running-clusterfuzzlite/github_actions.md:319]: Should this be "##" and the rest be "###" instead?

**Review by oliverchang — APPROVED** [review]

File: docs/running-clusterfuzzlite/github_actions.md (hunk 2 above)
Comment (jonathanmetzman) [reply docs/running-clusterfuzzlite/github_actions.md:319]: Right.

**Pull request merged**